README.md

```diff
@@ -1,6 +1,6 @@
 # Exercism Delphi Pascal Track
 
-[![Build Status](https://travis-ci.org/exercism/delphi.svg?branch=master)](https://travis-ci.org/exercism/delphi)
+![CI](https://github.com/exercism/delphi/workflows/configlet/badge.svg)
```

Comment:
@ErikSchierboom, is `/workflows/configlet/badge.svg` something that is coming?

Reply:
configlet is still using Travis at the moment, but I've [created an issue](https://github.com/exercism/configlet/issues/184) to get it converted to GitHub actions :)